features/verblijfplaats.feature

```diff
@@ -0,0 +1,140 @@
+# language: nl
+
+@proxy @post-assert
+Functionaliteit: leveren van een verblijfplaats
+
+  Rule: Een verblijfplaats is onbekend wanneer land (13.10) de waarde '0000' heeft
+    - veld 'type' krijgt de waarde 'VerblijfplaatsOnbekend' 
+    - veld 'verblijfplaatsOnbekend' wordt opgenomen met de boolean waarde true
+
+    Scenario: land onbekend 
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                | waarde   |
+      | gemeenteVanInschrijving (09.10)     | 1999     |
+      | datumInschrijvingInGemeente (09.20) | 20220128 |
+      | land (13.10)                        | 0000     |
+      | datumAanvangAdresBuitenland (13.20) | 20220128 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                             |
+      | type                | RaadpleegMetBurgerservicenummer    |
+      | burgerservicenummer | 555550001                          |
+      | fields              | burgerservicenummer,verblijfplaats |
+      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'verblijfplaats' gegevens
+      | naam                              | waarde                 |
+      | type                              | VerblijfplaatsOnbekend |
+      | verblijfplaatsOnbekend            | true                   |
+      | gemeenteVanInschrijving           | 1999                   |
```

Comment:
Hier is gemeenteVanInschrijving (ook in OAS specificatie) een string ipv een tabelwaarde. Is dit correct?

Reply:
zo is het nu wel gedefinieerd, maar dat is m.i. helemaal fout. Ik heb daarvoor #975 gemaakt. Is denk ik gevolg van overkoepelende gegevens niet (altijd) in abstract opnemen. Zie #956